src/extension.ts

```diff
@@ -119,9 +172,18 @@ export function activate(context: vscode.ExtensionContext): void {
 	context.subscriptions.push(openWithCommand);
 	context.subscriptions.push(telemetryReporter);
 	context.subscriptions.push(copyOffsetAsDec, copyOffsetAsHex);
+	context.subscriptions.push(compareSelectedCommand);
+	context.subscriptions.push(
+		vscode.workspace.registerFileSystemProvider("hexdiff", new HexDiffFSProvider(), {
+			isCaseSensitive: true,
```

Comment:
I think we'd want this to match the underlying filesystem's case sensitivity. Probably `isCaseSensitive: typeof process !== 'undefined' && process.platform !== 'win32' && process.platform !== 'darwin'` is an okay heuristic